x/payment/types/wirepayfordata_test.go

```diff
@@ -88,11 +84,6 @@ func TestWirePayForData_ValidateBasic(t *testing.T) {
 			msg:     invalidSquareSizeMsg,
 			wantErr: ErrCommittedSquareSizeNotPowOf2,
 		},
-		{
-			name:    "wrong but valid square size",
-			msg:     badSquareSizeMsg,
-			wantErr: ErrInvalidShareCommit,
-		},
```

Comment:
I'm not sure why this test fails now. Attempts to fix it result in a different error: `ErrInvalidShareCommitments` and not `ErrInvalidShareCommit`.

I don't _think_ this test is still necessary because we test for `ErrInvalidShareCommit` [here](https://github.com/celestiaorg/celestia-app/blob/main/x/payment/types/wirepayfordata_test.go#L82-L84) but curious for reviewers thoughts.

Reply:
I think this is just because the after we increase the share size, it makes the commitment to the message the same for different sizes since it can now fit on one row. If we change the test data to create valid pfd with 4000 byte msg instead of 2000, this test passes